packs/menace-under-otari-bestiary/black-kobold-dragon-mage-bb.json

```diff
@@ -910,7 +872,23 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "ac",
+                        "key": "RollOption",
+                        "option": "scamper",
+                        "toggleable": true
+                    },
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "scamper"
+                        ],
+                        "selector": "ac",
+                        "type": "circumstance",
+                        "value": 2
+                    }
```

Comment:
Should we add:
```
{
	"key": "FlatModifier",
	"predicate": [
		"scamper"
	],
	"selector": "land-speed",
	"value": 5
}
```
and change the RollOption domain to `all`?

Reply:
I would wager the 30 is actually an error, we'll check.